peripherals/adc/adc_l0.yaml

```diff
@@ -6,43 +6,43 @@ ADC:
       _read:
         NotComplete: [0, "Calibration is not complete"]
         Complete: [1, "Calibration complete"]
-      _write:
+      _W1C:
```

Comment:
Just out of curiosity, what does this change do? I am assuming w1c means "write 1 to clear", but what does it affect? :)

Reply:
Good question.
Yes, it annotates field with `modifiedWriteValues`.

For Rust it says to use [BitWriter1C](https://docs.rs/stm32f1-staging/latest/stm32f1_staging/type.BitWriter1C.html) instead of `BitWriter`.
Also it modifies constants in `Writable trait`(https://docs.rs/stm32f1-staging/latest/stm32f1_staging/trait.Writable.html#required-associated-consts) implementation to prevent touching this field on `write` and `modify`.
See https://docs.rs/stm32f1-staging/latest/src/stm32f1_staging/generic.rs.html#830

Reply:
Wow, interesting!